style.css

```diff
@@ -449,6 +449,125 @@ ul {
   display: none;
 }
 
+/* pop up menu classes */
+
+.header-pp {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  align-items: baseline;
+}
+
+.closeButton-class {
+  padding: 12px;
+  background-color: #fff;
+  background-image: url(./assets/icons/closeIcon.svg);
+  background-repeat: no-repeat;
+  float: right;
+}
+
+.pop-up-class {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  gap: 12px;
+  position: fixed;
+  width: 343px;
+  height: 713px;
+  left: calc(50% - 343px / 2);
+  top: 20px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin: 0;
+}
+
+.project-head {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.project-descrip {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.desktop-button-pp {
+  display: none;
+}
+
+.mobile-button-pp {
+  display: block;
+}
+
+.btn-container-class-pp {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.btn-seelive-pp {
+  background-image: url(./assets/images/see-live.svg);
+  background-position: 85% center;
+  background-repeat: no-repeat;
+  display: flex;
+  justify-content: flex-start;
+  width: 142px;
+  padding: 12px 16px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  gap: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.btn-seesource-pp {
+  background-image: url(./assets/images/see-source.png);
+  background-position: 85% center;
+  background-repeat: no-repeat;
+  display: flex;
+  justify-content: flex-start;
+  width: 142px;
+  padding: 12px 16px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  gap: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.lang-container-pp {
+  display: flex;
+  gap: 8px;
+}
+
+.languages-pp {
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 6px;
+}
+
 @media screen and (min-width: 768px) {
```

Comment:
Please make sure that the popup is responsive, and that it fits mobile and desktop sizes properly. This is what happens to your popup around the viewpoint 746:

![image](https://user-images.githubusercontent.com/38088818/174057357-9dc44e5e-9d99-426c-88ff-a3b4df45c9a7.png)

- Kindly note that the background content of the main page is also visible. Please make sure that the background content is always blurry on any viewpoint to make sure that your popup stands out as per FIGMA design specifications.

![image](https://user-images.githubusercontent.com/38088818/174057850-59f0233b-28b5-49ff-a3a7-104e6cd34b0e.png)